src/game/models/asteroid.rs

```diff
@@ -1,5 +1,16 @@
 //! Defines the asteroid component.
-
+//!
+//! Asteroids are shapes that randomly float around the screen.
+//! They have several properties:
+//! * `pos`: the asteroid's position
```

Comment:
These could be defined here or on the actual `Asteroid` struct itself. Not sure it's a big deal either way but just wanted to throw it out.

In intellij-rust you can Ctrl-Q on a struct or member and it will pop up rendered Markdown which is kinda nice.